src/main/kotlin/no/nav/medlemskap/sykepenger/lytter/config/Configuration.kt

```diff
@@ -34,7 +34,7 @@ private val defaultProperties = ConfigurationMap(
         "KAFKA_CREDSTORE_PASSWORD" to "changeme",
         "KAFKA_ENABLED" to "false",
         "BRUKERSPORSMAAL_ENABLED" to "Ja",
-        "AZURE_APP_TENANT_ID" to "",
+        "AZURE_APP_TENANT_ID" to "966ac572-f5b7-4bbe-aa88-c76419c0f851",
```

Comment:
```suggestion
        "AZURE_APP_TENANT_ID" to "",
```

Er dette en nøkkel?

"Er bare i dev" -> tenker det fortsatt er verdt å ikke pushe det, plutselig glemmer man seg og pusher en nøkkel til et dev-miljø som bruker maskerte/skarpe data (og ikke bare syntetiske).
Mange nok slike om dagen mens fss fortsatt er en ting.

Reply:
dett er en identifikasjon på applikasjon og er ikke sensitiv info. Denne må deles når man eksempelvis skal hente ut et token mot en annen tjeneste en sin egen.. Kan fjerne den da den ligger embedded på server, men det betyr at jeg ikke kan kjøre kode lokalt. frem til vi løser lokalt nais miljø så lar jeg den bli som i dag